src/spirc.rs

```diff
@@ -246,7 +295,9 @@ impl SpircTask {
 
         if frame.get_ident() == self.ident ||
            (frame.get_recipient().len() > 0 && !frame.get_recipient().contains(&self.ident)) {
-            return;
+            if frame.get_typ() == MessageType::kMessageTypeNotify {
```

Comment:
I'm not arguing librespot shouldn't be able to control itself, just that there's no need to broadcast a message out.
If librespot is currently active we can just process the command locally.